contracts/RedstoneCoreProxy.sol

```diff
@@ -22,50 +22,44 @@ abstract contract RedstoneCoreProxy is Proxy {
      */
     function _delegate(address implementation) internal override {
 
-        //Check if transaction contains Limestone marker
+        // Check if transaction contains Redstone marker
         bool isTxWithPricing = false;
         if (msg.data.length > 32) {
             isTxWithPricing = msg.data.toBytes32(msg.data.length - 32) == MARKER;
```

Comment:
Let's intrroduce 2 constants:
MARKER_SLOT_SIZE = 32
LEN_SLOT_SIZE = 2
To improve readibility